nautobot/core/templates/admin/base.html

```diff
@@ -95,10 +47,10 @@
     </style>
 </head>
 <body {% if is_popup %}style="padding-top: 0px;"{% endif %}>
-{% if not is_popup %}
-    {% include 'inc/nav_menu.html' %}
-{% endif %}
-<div class="container-fluid wrapper" {% if is_popup %}style="padding-bottom: 0px;"{% endif %}>
+
+{% include 'inc/nav_menu.html' %}
+
+<div class="container-fluid wrapper" id="main-content" {% if is_popup %}style="padding-bottom: 0px;"{% endif %}>
```

Comment:
The admin UI uses pop-up windows for some sub-forms within other forms. Do we still need the `is_popup` conditional here? I wouldn't think we'd want the nav menu to appear in those pop-ups.

Reply:
Yeah, I think we should add that conditional back in:

![image](https://github.com/nautobot/nautobot/assets/5603551/aeb20f48-ddd7-404e-96f8-b0c624724b44)

Reply:
Nav menu conditional still needs to be re-added.